install/configuring_inventory_file.adoc

```diff
@@ -246,13 +250,13 @@ on an Amazon Web Services (AWS) cluster. The default value is `false`.
 
 |`template_service_broker_selector`
 |Default node selector for automatically deploying template service broker pods,
-defaults `{"region": "infra"}`. See
+defaults `{"node-role.kubernetes.io/infra":"true"}`. See
 xref:configuring-node-host-labels[Configuring Node Host Labels] for details.
 
 |`osm_default_node_selector`
 |This variable overrides the node selector that projects will use by default when
 placing pods, which is defined by the `projectConfig.defaultNodeSelector` field
-in the master configuration file. This defaults
+in the master configuration file. Starting in {product-title} 3.9, this defaults
 to `node-role.kubernetes.io/compute=true` if undefined.
```

Comment:
@bysnupy The _Starting in {Product-title} 3.9_ was removed through https://github.com/openshift/openshift-docs/commit/4168b2abe0c3e08caa43dee903ace30d9bba67bf#diff-80f7c8ee1abcb1ceb42d8350b49bf604. 

The sentence starts with _This defaults..._

Reply:
Fixing here: https://github.com/openshift/openshift-docs/pull/12510